From e1175ca83080bb992bfb93ec0467cb040565931e Mon Sep 17 00:00:00 2001
From: "Steven Rostedt (Red Hat)" <rostedt@goodmis.org>
Date: Thu, 20 Oct 2016 18:31:27 -0400
Subject: [PATCH 088/134] trace-view: Place focus back into search box after
 search

When searching the tree rows, after an item is found, it would be nice to
repeat the search with a keyboard instead of having to move the mouse back
to the entry and selecting again. Instead, have the focus come back to the
search entry box and allow the search to happen again with just hitting
"enter".

Signed-off-by: Steven Rostedt <rostedt@goodmis.org>
---
 trace-view.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/trace-view.c b/trace-view.c
index 3b95529d98c3..2a5b1ad5f2c8 100644
--- a/trace-view.c
+++ b/trace-view.c
@@ -864,6 +864,8 @@ static void search_tree(gpointer data)
 	path = gtk_tree_model_get_path(model, &iter);
 	select_row_from_path(info->treeview, path);
 	gtk_tree_path_free(path);
+
+	gtk_widget_grab_focus(GTK_WIDGET(info->entry));
 }
 
 void trace_view_search_setup(GtkBox *box, GtkTreeView *treeview)
-- 
2.5.5

